protocol/encryption/encryptor.go

```diff
@@ -592,6 +596,132 @@ func (s *encryptor) EncryptPayload(theirIdentityKey *ecdsa.PublicKey, myIdentity
 	return response, targetedInstallations, nil
 }
 
+func (s *encryptor) getNextKeyID(communityId string) (uint32, error) {
```

Comment:
This name seems to be a bit generic, I would maybe rename to `getNextHashRatchetKeyID` or similar